src/content.js

```diff
@@ -551,6 +551,25 @@ function init() {
 			event.preventDefault();
 			indentInput(event.target);
 			return false;
+		} else if (event.which === 13 && event.shiftKey && (event.ctrlKey || event.metaKey)) {
+			const singleCommentButton = event.target.closest('.js-inline-comment-form')
+					.querySelector('.review-simple-reply-button');
+
+			if (singleCommentButton) {
+				event.preventDefault();
+				singleCommentButton.click();
+				return false;
+			}
+		} else if (event.which === 27) {
+			const form = event.target.closest('.js-inline-comment-form'),
+				textarea = form.querySelector('.review-simple-reply-button'),
+				cancelButton = form.querySelector('.js-hide-inline-comment-form');
+
+			if (textarea && textarea.value !== '' && cancelButton) {
+				event.preventDefault();
```

Comment:
I think we can skip all of these `preventDefault`s because jQuery should interpret `return false` that way already.

Reply:
Okay.  I was copying the tab handling code above.